lib/ansible/modules/packaging/os/package.py

```diff
@@ -28,6 +28,7 @@
   name:
     description:
       - "Package name, or package specifier with version, like C(name-1.0)."
+      - This can be a package name, or with most package managers, a file path to a package or a URL.
```

Comment:
May be worth adding "From Ansible 2.5 this can also be..."

Reply:
Well, it's supported URLs/paths for Yum (at least) for longer...not sure how much detail we should squeeze in there.